strings/traumas.json

```diff
@@ -65,7 +65,9 @@
         "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
         "who the HELL do u thenk u r?!!!!",
         "without oxigen blob don't evoluate?",
-        "youed call her a toeugh bithc"
+        "youed call her a toeugh bithc",
+		"built DIFFERENT. I Stack Paper!",
+		"MAKE JIANT CRISTAL BURN I KEPT THROWING WELDERS IT WOOD LIGHT"
```

Comment:
```suggestion
        "youed call her a toeugh bithc",
	"built DIFFERENT. I Stack Paper!",
	"MAKE JIANT CRISTAL BURN I KEPT THROWING WELDERS IT WOOD LIGHT",
```

Reply:
![image](https://github.com/tgstation/tgstation/assets/7019927/66754e2b-89df-48be-aed3-5ae2af5edd59)
???????

Reply:
I fixed it but what the fuck